test/unit/specs/components/Personalization/helper/dom/createFragment.spec.js

```diff
@@ -12,3 +12,11 @@ governing permissions and limitations under the License.
 
 // eslint-disable-next-line no-unused-vars
```

Comment:
Can you remove this comment?  It's no longer needed for the linter because you use createFragment now.